contrib/seccomp/seccomp_default.go

```diff
@@ -399,7 +399,7 @@ func DefaultProfile(sp *specs.Spec) *specs.LinuxSeccomp {
 			Action: specs.ActAllow,
 			Args:   []specs.LinuxSeccompArg{},
 		})
-	case "amd", "x32":
+	case "amd64":
```

Comment:
what about 32 here?

Reply:
the syscall is only available on 64-bit (https://www.systutorials.com/docs/linux/man/2-arch_prctl/) so I assume the `x32` was an error anyway.

Reply:
from my digging around does not exist as a valid golang runtime.GOARCH response..